### PR TITLE
35 rename dimension

### LINKED ISSUE #35: Rename the "points" dimension to space dimension
stmtools now uses the label "points" for the space dimension. A more generic and relevant label for that would be "space".

### PR DESCRIPTION
**Pull request by rogerkuou**
Fix #35

### CODE REVIEW

**Comment by rogerkuou** [comment]
Hi @SarahAlidoost, could you please review this PR? Thanks!

File: tests/test_util.py [hunk 1]
```diff
@@ -3,28 +3,7 @@
...
-                da.arange(npoints * ntime).reshape((npoints, ntime)),
-            ),
-        ),
-        coords=dict(
-            lon=(["points"], da.arange(npoints)),
-            lat=(["points"], da.arange(npoints)),
-            time=(["time"], np.arange(ntime)),
-        ),
-    ).unify_chunks()
+from .test_stm import stmat
```
Comment (SarahAlidoost) [inline tests/test_util.py:6]:
```suggestion
from tests.test_stm import stmat
```

Comment (SarahAlidoost) [reply tests/test_util.py:6]: Pytest returns an error with relative import, so I suggest here adding `tests.`, can you check it?

Comment (rogerkuou) [reply tests/test_util.py:6]: I found `tests.` also works, will fix this

**Review by SarahAlidoost — APPROVED** [review]
@rogerkuou thanks, it looks good. Just a comment about the `import` in the test, see my suggestion. When it is fixed, feel free to merge it :+1:

**Pull request merged**